presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -1651,6 +1652,10 @@ public String getValidWriteIds(HiveIdentity identity, List<SchemaTableName> tabl
                         }
                     });
         }
+        catch (ConfigValSecurityException e) {
```

Comment:
Is it possible to have a test for that?

Reply:
It's tested by running Hive ACID with Hive <3 metastore.